app/src/main/java/com/ably/tracking/demo/publisher/ably/DefaultAssetTracker.kt

```diff
@@ -86,6 +89,11 @@ class DefaultAssetTracker(
             ?.launchIn(coroutineScope)
     }
 
+    private fun authorize(): String =
```

Comment:
IMO the name is very generic and does not suggest that it is returning something :thinking: I'd change it to something like `getAuthToken` or `createAuthToken` :wink:

Reply:
https://github.com/ably/asset-tracking-android-rider-app-demo/pull/39/commits/bdafac00ef69089740b19cdd4d63ca5732750780